onnxscript/ir/_convenience.py

```diff
@@ -395,3 +395,44 @@ def create_value_mapping(graph: _core.Graph) -> dict[str, _core.Value]:
                 continue
             values[value.name] = value
     return values
+
+
+def replace_nodes_and_values(
+    graph_or_function: _core.Graph | _core.Function,
```

Comment:
```suggestion
    graph_or_function: _core.Graph | _core.Function,
    /,
```

Reply:
Done (but not pushed yet). But is there a specific principle being used here, to distinguish between the two classes of parameters? Except for the name, which is admittedly not great (we could do with a better alternative to graph_or_function ... how does `container` sound? or `graph_like` ?)

Reply:
I like `graph_like`!

> is there a specific principle being used here, to distinguish between the two classes of parameters

not sure what you mean? could you say more?

Reply:
I mean, why do you prefer that graph_or_function be positional, while the others are allowed to be positional or by keyword?